extensions/wikia/SpecialInterwikiEdit/SpecialInterwikiEdit.php

```diff
@@ -370,7 +370,7 @@ function updateAction(action){
 	if ($from) $from = "WHERE iw_prefix like '". $db->escapeLike($from) . "%'";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)